.github/workflows/run-fvt.yml

```diff
@@ -20,14 +20,16 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.18.7'
-      - name: Setup Minikube
-        run: |
-          wget --no-verbose https://github.com/kubernetes/minikube/releases/download/v1.25.1/minikube-linux-amd64
-          sudo cp minikube-linux-amd64 /usr/local/bin/minikube
-          sudo chmod 755 /usr/local/bin/minikube
-          sudo apt-get install -y conntrack socat
-          minikube start --driver=none --kubernetes-version v1.22.10
+          go-version: '1.18.7'            
+      - name: Start Minikube
+        uses: medyagh/setup-minikube@v0.0.11
+        id: minikube
+        with:
+          minikube-version: 1.27.1
+          container-runtime: docker
```

Comment:
Since you change the way we install minikube, some of the action steps no longer work as intended ("Free up disk space", "Check installation"). You will need to add `eval $(minikube -p minikube docker-env)` there as well. These steps not working might account for some of the disk storage issues you encountered during the FVT go install.

We used to start minikube with `--driver=none`. I think after your changes minikube will have to auto-detect the driver. Did you try `driver: none`? There may be [more to configure for K8s 1.25](https://minikube.sigs.k8s.io/docs/drivers/none/#requirements) though.

Reply:
Yea I tried to use none driver but it was not working well and I found [this issue](https://github.com/kubernetes/minikube/issues/14403#issuecomment-1164706914). For this reason, I use the docker driver.

Reply:
I do see an extra line now but Github still complains about the missing new line at the end of file:

![image](https://user-images.githubusercontent.com/12246093/232176181-530241df-3db6-4e29-aa47-5a0bca3211f9.png)


![image](https://user-images.githubusercontent.com/12246093/232176163-7b0089dc-9b52-48c3-ab77-89b06ccf75bc.png)

Are you working on Windows by any chance?

https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log